### PR TITLE
Update .NET Framework PreReq for new Windows versions

### PR DESCRIPTION
**Pull request by ojrb**
This PR Solved the issue #1694, I add the new .Net Framework preReq for new versions and clarify the Non-HTTP Activation only for old Windows versions, please review and merge

### CODE REVIEW

**Comment by illfated** [comment]
Rich Text Diff view for easier view of effective changes:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4781/files?short_path=4f6bbe8#diff-4f6bbe86a8d15151cd3a9cf0d3d61999

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: mdop/mbam-v25/mbam-25-server-prerequisites-for-stand-alone-and-configuration-manager-integration-topologies.md [hunk 1]
```diff
@@ -273,7 +274,7 @@ The following table lists the installation prerequisites for the MBAM Administra
 <li><p><strong>WCF Activation</strong></p>
 <ul>
 <li><p>HTTP Activation</p></li>
-<li><p>Non-HTTP Activation</p>
+<li><p>Non-HTTP Activation (Only for Windows Server 2008, 2012 or 2012 R2)</p>
```
Comment (JohanFreelancer9) [inline mdop/mbam-v25/mbam-25-server-prerequisites-for-stand-alone-and-configuration-manager-integration-topologies.md]:
```suggestion
<li><p>Non-HTTP Activation (Only for Windows Server 2008, 2012, and 2012 R2)</p>
```

**Comment by ojrb** [comment]
Accepted the copy/edit suggestions

**Comment by e0i** [comment]
@neonix Pending your input for copy review. Thanks.

**Comment by illfated** [comment]
I noticed that Nicole (nenonix) has been quiet for a couple of weeks.
Could be on vacation, but I don't know until she comments again.

**Comment by e0i** [comment]
@dansimp The PR updates the "Windows Server Features" section that solves issue #1694 is available for review. Thanks.

**Pull request merged**